sdk/src/quotes/public/increase-liquidity-quote.ts

```diff
@@ -59,7 +61,7 @@ export type IncreaseLiquidityQuote = IncreaseLiquidityInput & { tokenEstA: BN; t
  * @param whirlpool - A Whirlpool helper class to help interact with the Whirlpool account.
  * @returns An IncreaseLiquidityInput object detailing the required token amounts & liquidity values to use when calling increase-liquidity-ix.
  */
-export function increaseLiquidityQuoteByInputToken(
+export function increaseLiquidityQuoteByInputToken_PriceSlippage(
```

Comment:
I think SDK uses lower camel case for names.
Is there strong reason to use underscore here ?
One thing came to mind is `UsingPriceSlippage`.